generator/counter.go

```diff
@@ -11,7 +14,8 @@ type Counter struct {
 // NewCounter returns new generator for growing points. Starting value is an increment as well.
 // Possibly it can randomize values around increment.
 // When deviation is set, it the next value won't be less then previous.
-func NewCounter(name string, start, stop, step uint, randomizeStart bool, value, deviation float64) *Counter {
+func NewCounter(name string, start, stop, step uint, randomizeStart bool, value, deviation float64) (*Counter, error) {
+
```

Comment:
Why empty lines in the beginning and ending of functions?

Reply:
I have fixed it